idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -334,83 +344,74 @@ def _make_params(self):
 
     def _make_performance_method(self):
 
-        solvent_list = self.config.hot_side.property_package.component_list_solvent
-
-        def rule_trh(blk, t):
-            return (blk.hot_side.properties_out[t].temperature /
-                    blk.hot_side.properties_in[t].temperature)
-        self.trh = Expression(self.flowsheet().time, rule=rule_trh,
-                              doc='Ratio of hot outlet temperature to hot'
-                                  'inlet temperature')
-
-        def rule_trc(blk, t):
-            return (blk.cold_side.properties_out[t].temperature /
-                    blk.cold_side.properties_in[t].temperature)
-        self.trc = Expression(self.flowsheet().time, rule=rule_trc,
-                              doc='Ratio of cold outlet temperature to cold'
-                                  ' inlet temperature')
-
-        def rule_cp_comp_hot(blk, t, j):
-            return 1e3 * (
-                blk.hot_side.properties_in[t]._params.cp_param[j, 1] +
-                blk.hot_side.properties_in[t]._params.cp_param[j, 2] / 2 *
-                blk.hot_side.properties_in[t].temperature *
-                (blk.trh[t] + 1) +
-                blk.hot_side.properties_in[t]._params.cp_param[j, 3] / 3 *
-                (blk.hot_side.properties_in[t].temperature**2) *
-                (blk.trh[t]**2 + blk.trh[t] + 1) +
-                blk.hot_side.properties_in[t]._params.cp_param[j, 4] / 4 *
-                (blk.hot_side.properties_in[t].temperature**3) *
-                (blk.trh[t] + 1) * (blk.trh[t]**2 + 1) +
-                blk.hot_side.properties_in[t]._params.cp_param[j, 5] / 5 *
-                (blk.hot_side.properties_in[t].temperature**4) *
-                (blk.trh[t]**4 + blk.trh[t]**3 + blk.trh[t]**2 + blk.trh[t] + 1))
-
-        self.cp_comp_hot = Expression(self.flowsheet().time,
-                                      solvent_list,
-                                      rule=rule_cp_comp_hot,
-                                      doc='Component mean specific heat capacity'
-                                      ' btw inlet and outlet'
-                                      ' of hot-side temperature')
+        solvent_list = \
+            self.config.hot_side.property_package.solvent_set
+
+        def rule_mass_frac_solvent_hot(blk, t, j):
+            return blk.hot_fluid.properties_in[t].mass_frac_phase_comp['Liq', j] /\
```

Comment:
What exactly is this meant to be? Specifically, what is the correct value if `j` is not a solvent? Should this return `Expression.Skip` if `j` is not a solvent?